py/desispec/workflow/procfuncs.py

```diff
@@ -348,25 +351,43 @@ def create_batch_script(prow, queue='realtime', dry_run=0, joint=False, system_n
             else:
                 scriptpathname = scripts[0]
     else:
-        if joint:
-            cmd = desi_proc_joint_fit_command(prow, queue=queue)
-        else:
-            cmd = desi_proc_command(prow, queue=queue)
-        scriptpathname = batch_script_name(prow)
+        if prow['JOBDESC'] != 'tilenight':
```

Comment:
@akremin thanks for catching this. Since I designed `create_desi_proc_tilenight_batch_script` in such a way that the code to generate the `desi_proc_tilenight` command for batch scripts is local to that function, it doesn't make sense to report that command here. I changed 
```
            log.info("Command to be run: {}".format(cmd.split()))
```
to
```
            if prow['JOBDESC'] != 'tilenight':
                log.info("Command to be run: {}".format(cmd.split()))
```
to avoid the crashing you mentioned.

Reply:
@akremin @sbailey to clarify; printing the command line(s) in the batch file for `dry_run>1` is a desirable feature that tilenight batch scripts do not currently have, which was requested above. It is not clear to me how to do this in a consistent way without creating a new function whose only purpose is to create the command line for `desi_proc_tilenight` arguments, which I don't think is a good idea. Let's address how to do this after merging this PR.